helper/helper_test.go

```diff
@@ -98,4 +265,235 @@ var _ = Describe("helper tests", func() {
 			Expect(isSubset).To(Equal(false))
 		})
 	})
+
+	Describe("doRestoreAgent Mocked unit tests", func() {
+		BeforeEach(func() {
+			// Setup mocked tests environment
+			*singleDataFile = false
+			*content = 1
+			*oidFile = "testoid.dat"
+			*isResizeRestore = true
+			*origSize = 5
+			*destSize = 3
+			*pipeFile = "mock"
+			*onErrorContinue = true
+		})
+		It("successfully restores using a single data file when inputs are valid and no errors occur", func() {
+			*singleDataFile = true
+
+			oidBatch := []oidWithBatch{{oid: 1, batch: 1}}
+			steps := []helperTestStep{
+				{},
+				{getRestorePipeWriterArgExpect: "mock_1_1", getRestorePipeWriterResult: true, checkSkipFileArgTableOid: 1, checkSkipFileResult: false},
```

Comment:
Why do you use named arguments in some tests and not in others?

Reply:
question is still relevant

Reply:
Both variants are valid syntax. Here there is only one step and it's more readable with named fields. At the other places there are more steps and to avoid a mess with a lot of repeated field names shorter form is accessible. Test writer is free to choose one form he like more.